packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -225,7 +251,7 @@ const FinancialAssessment = ({
             setIsLoading(false);
             history.push(routes.personal_details);
         } else {
-            WS.authorized.storage.getFinancialAssessment().then(data => {
+            WS.authorized.storage.getFinancialAssessment().then((data: TFinancialAssessementData) => {
```

Comment:
```suggestion
            WS.authorized.storage.getFinancialAssessment().then((data: SetFinancialAssessmentResponse) => {
```

Reply:
There is no property called *.error.message. So I created a type
`type TFinancialAssessementData = Pick<SetFinancialAssessmentRequest, 'financial_information'> &
    Record<string, { [key: string]: React.ReactNode }>;`
Also we just need the `financial_information` section of the response.

Reply:
@likhith-deriv You can use `in` operator for that, Let's not create a custom type and just use `SetFinancialAssessmentResponse` as later we will slowly get rid of types in code as everything will infer 🙏🏻 

```ts
if (error && 'message' in error) {
  ...
}
```